src/python/WMCore/DataStructs/LumiList.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, filename = None, lumis = None, runsAndLumis = None, runs = No
                 run = str(run)
                 if run not in runsAndLumis:
                     runsAndLumis[run] = []
-                runsAndLumis[run].append(lumi)
+                runsAndLumis[run].extend(lumi)
```

Comment:
The list of lists is not a list of lumis, it is a list of ranges. So if you need to do this, you need to extend the list *outside* of this class.

In other words [[1,5], [8,9]] is equivalent to [1,2,3,4,5,8,9]